src/components/CollapsibleTable.js

```diff
@@ -60,6 +66,68 @@ class CollapsibleTable extends Component {
     this.setState({showServerDetailsModal: true, activeRowData: rowData});
   }
 
+  handleShowMenuServerDetails = () => {
```

Comment:
it occurs to me reading this that this component is very server list specific, but thats a pre-existing condition and not something introduced here

Reply:
see your point. maybe next round to cleanup.